src/core/net/ip6_address.cpp

```diff
@@ -74,6 +75,18 @@ bool Prefix::IsLinkLocal(void) const
            ((mPrefix.mFields.m16[0] & BigEndian::HostSwap16(0xffc0)) == BigEndian::HostSwap16(0xfe80));
 }
 
+bool Prefix::IsRequiredAnycast(void) const
+{
+    for (int i = mLength / 8; i < OTBR_IP6_ADDRESS_SIZE; ++i)
```

Comment:
What if we have an address like `2001:db8:daed:b008::/63`

Please also include test for this.